lib/ansible/module_utils/common/network.py

```diff
@@ -9,8 +9,8 @@
 from struct import pack
 from socket import inet_ntoa
 
-from ansible.module_utils.six.moves import zip
-
+# backwards compat
+zip = zip  # pylint: disable=self-assigning-variable
```

Comment:
Would this be cleaner?
```suggestion
from builtins import zip
```

Reply:
👍🏻